src/target.d

```diff
@@ -153,7 +153,12 @@ struct Target
      */
     extern (C++) static uint fieldalign(Type type)
     {
-        return type.alignsize();
+        const size = type.alignsize();
+
+        if ((global.params.is64bit || global.params.isOSX) && (size == 16 || size == 32))
+            return size;
+
+        return (8 < size) ? 8 : size;
```

Comment:
Lol, thanks for the hint (LLVM obviously does), I've just seen that I messed up a front-end merge, so that we now have 2 `Target::alignsize()` implementations in LDC (D and C++), don't know which one gets used. ;)